storage/remote/codec.go

```diff
@@ -625,9 +625,11 @@ func exemplarProtoToExemplar(ep prompb.Exemplar) exemplar.Exemplar {
 }
 
 // HistogramProtoToHistogram extracts a (normal integer) Histogram from the
-// provided proto message. The caller has to make sure that the proto message
-// represents an integer histogram and not a float histogram.
+// provided proto message.
```

Comment:
I would leave the old comment in here, because the caller still has to make sure. Maybe add "Otherwise, the function panics."